packages/wallet-stamper/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@turnkey/wallet-stamper",
-  "version": "0.0.5",
+  "version": "1.0.0",
```

Comment:
should we make this initial launch as beta? i.e. `0.1.0`? or are we comfortable full sending as `1.0.0`? I'm fine with either; if we're feeling good about the integration with demo wallet, let's do `1.0.0` :)

Reply:
Yeah I was on the fence between beta/major version, I decided the API feels solidified -- I have some ideas for future improvements but they change the core API